store.js

```diff
@@ -88,17 +88,23 @@ class MongoStore extends Store {
    * @memberOf MongoStore
    */
   removeById(req) {
+    var id
+    try {
+      id = this.ObjectID(req.id)
```

Comment:
This kind of check is really bad. We can't trust in that an exception is thrown. Please check the format via regex and swap it out in a separate function to avoid duplicate code.

Reply:
[This](https://stackoverflow.com/a/25497438/1893773) is even better 

```js
var mongodb = require("mongodb"),
    objectid = mongodb.BSONPure.ObjectID;

console.log(objectid.isValid('53fbf4615c3b9f41c381b6a3'));
```